src/pages/experimental/create-flows/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create flows
+title: Create flow pattern
```

Comment:
```suggestion
title: Create flows
```

Let's keep the title as is just to maintain consistent nomenclature on our side nav. We're working on a better nav bar that will allow more complete titles.

Reply:
Ok, that's fine by me - Vikki mentioned there migth be abit of a reorganisation going on with the carbon site - we'll be super keen to hear about that @connor-leech